.azure-pipelines/client.yml

```diff
@@ -83,6 +109,7 @@ jobs:
         os.name: 'MacOS'
```

Comment:
Sorry if we are hijacking your PR but if you are trying to make this file match the guidelines can you also update these matrix strategy variables?

Reply:
Haha, yes I just pushed a commit for the same